app/views/checkout/_tabs.html.haml

```diff
@@ -1,4 +1,5 @@
-.flex
+-# Prevent Turbo pre-fetch which changes cart state
+.flex{'data-turbo': "false"}
```

Comment:
could you just disable the pre-fetch ? 
```suggestion
.flex{'data-turbo-prefetch': "false"}
```

Reply:
Thanks, I didn't know about [all the options](https://turbo.hotwired.dev/reference/attributes) actually!
That seems better: then we still get consistent behaviour when navigating.